rebar.config

```diff
@@ -1,5 +1,5 @@
 {deps, [
-    {gpb, ".*", {git, "https://github.com/HalloAppInc/gpb.git", {tag, "h1.2"}}}
+            {gpb, ".*", {git, "https://github.com/HalloAppInc/gpb.git", {branch, "merge-upstream"}}}
```

Comment:
dont think these are linked. get rid of gpb changes here - i would prefer we upgrade gpb completely separately.